### PR TITLE
helm chart improvements 

### PR DESCRIPTION
**Pull request by achraf-mer**

### CODE REVIEW

File: helm/h2ogpt-chart/values.yaml [hunk 1]
```diff
@@ -86,15 +94,29 @@ vllm:
     repository: gcr.io/vorvan/h2oai/h2ogpt-runtime
     pullPolicy: IfNotPresent
 
+  imagePullSecrets:
+    - name: docker-creds
```
Comment (EshamAaqib) [inline helm/h2ogpt-chart/values.yaml:98]: I think would be better to leave this field empty

File: helm/h2ogpt-chart/values.yaml [hunk 2]
```diff
@@ -86,15 +94,29 @@ vllm:
...
+
   storage:
     size: 512Gi
     class: ebs-csi
   
   overrideConfig:
+    CUDA_VISIBLE_DEVICES: "0,1"
```
Comment (EshamAaqib) [inline helm/h2ogpt-chart/values.yaml:105]: Same here

File: helm/h2ogpt-chart/values.yaml [hunk 3]
```diff
@@ -112,8 +134,15 @@ vllm:
     seccompProfile:
 
   resources:
+
   nodeSelector:
```
Comment (EshamAaqib) [inline helm/h2ogpt-chart/values.yaml:138]: Same here

File: helm/h2ogpt-chart/values.yaml [hunk 4]
```diff
@@ -112,8 +134,15 @@ vllm:
...
+    hac.h2o.ai/accelerator: training-gpu
+
   tolerations:
```
Comment (EshamAaqib) [inline helm/h2ogpt-chart/values.yaml:141]: Same here

**Comment by EshamAaqib** [comment]
IMO, overrideConfig, containerArgs, nodeSelector and tolerations should be left empty so it configurable during installation

**Comment by achraf-mer** [comment]
> IMO, overrideConfig, containerArgs, nodeSelector and tolerations should be left empty so it configurable during installation

done, PTAL thanks.

**Review by pseudotensor — APPROVED** [review]

**Pull request merged**